lib/senders/index.js

```diff
@@ -96,16 +96,35 @@ module.exports = function (log, config, error, db, translator, sender) {
 
     var getSafeMailer = BOUNCES_ENABLED ? bounceGatedMailer : noopMailer
 
+    function getVerifiedSecondaryEmails(emailData) {
+      return emailData.reduce(function(list, email) {
+        if (! email.isPrimary && email.isVerified) {
+          list.push(email.email)
+        }
+        return list
+      }, [])
+    }
+
+    function getSecondaryEmails(emailData) {
+      return emailData.reduce(function(list, email) {
+        if (! email.isPrimary) {
+          list.push(email.email)
+        }
+        return list
+      }, [])
+    }
```

Comment:
I don't see anything in the tests that properly exercises these two functions. Should we tweak some of the existing tests so they pass in the different email types here and assert that whatever is supposed to happen happens?